media-video/droidcam/files/droidcam-modprobe.conf

```diff
@@ -0,0 +1,3 @@
+# Suggested values: 640×480, 960×720, 1280×720 (720p), 1920×1080 (1080p).
+options v4l2loopback-dc width=640 height=480
+
```

Comment:
The package already installs `/etc/modprobe.d/v4l2loopback-dc.conf` so this might be doing it double.

Reply:
The build installs a file called droidcam.conf under modules-load.d which loads the modules on boot, see README.gentoo.